app.go

```diff
@@ -484,10 +490,13 @@ func (app *App) Listen(address interface{}, tlsconfig ...*tls.Config) error {
 	app.init()
 	// Start prefork
 	if app.Settings.Prefork {
+		if app.Settings.Network == "ipv6" || isIPv6(addr) {
```

Comment:
The defaultNetwork is `tcp`,  maybe `tcp6` is better than `ipv6`.

Reply:
`tcp6` makes sense

Reply:
Made a typo, should be `tcp6` 👍